README.md

```diff
@@ -1,8 +1,28 @@
-# Sleep
+# Commute
 
-## Team members
+| Name  |  GitHub Username |
+|---|---|
+| Aditya Sharma  |   |
+|  Birinder Singh   | [birinder1469](https://github.com/Birinder1469)   |
+|  Daniel Lin   |  [danielglin](https://github.com/danielglin)  |
+|  Yenan Zhang   |    |
 
-- Yenan Zhang
-- Birinder Singh
-- Daniel Lin
-- Aditya Sharma
+
+## Plan of statistical analysis
+
+In this survey, we would test whether the average MDS enjoyment level for students living on and off-campus are different using hypothesis testing.
+
+Our null hypothesis is that there are no difference in enjoyment level between the two groups, our alternative hypothesis is that the two
+group means are different. Tests are conducted at 5% significance level.
+
+## Other survey questions to identify confounding variables
+
+The following are other questions we will include in our survey to identify confounding variables:
+
+- *How many minutes does your commute take?*  We are including this question because a student may have chosen to live on campus to avoid a long commute.
```

Comment:
We can be more specific here and ask for the time to commute one-way?

Reply:
I changed the question to specify the one-way commute time.